src/ntimporters/trello/trello_api.py

```diff
@@ -0,0 +1,128 @@
+""" Simple trello REST API client """
+import datetime
+import functools
+
+import requests
+
+# board -> project
+# list ->  section
+# card -> task
+# comment -> comment
+# checklist -> comment
+
+
+class TrelloClient:
```

Comment:
have you considered using `py-trello`? https://pypi.org/project/py-trello/

Reply:
yes, but:
- im getting only small subset of api
- young release (0.12?)
- unofficial package
so i decided to write it by myself.